### ISSUE TITLE
Copied contents contains unneccessary HTML tags and styling

### ISSUE THREAD

**Issue by NiklasBr** [a85/POSTMan-Chrome-Extension#8101]
**Describe the bug**

Copying text also copies lots of HTML cruft

**To Reproduce**
1. Go to the app
2. Click on any input field
3. Copy its content
4. Past into 

**Expected behavior**
Only text should be copied, it makes no sense to include a set of styles <span>:s when I copy the URL to a request or its headers

**Example**
If I copy from the GET/POST/PUT... URL input, this is what I get when I paste it into any other app such as a markdown editor or TinyMCE field in a web page or a Slack chat:

`<span style="color: rgb(80, 80, 80); font-family: OpenSans, Helvetica, Arial, sans-serif; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: pre-wrap; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">https://endpoint.example/rest/v1/articles/read?market=se&amp;lastmodified=2020-02-25T14:55:00</span>`

**App information (please complete the following information):**
 - App Type Electron
 - Postman Version You are up to date! Postman v7.19.1 is the latest version.
 - OS: Windows 10 and Ubuntu 18.04.04

**Comment by harryi3t** [comment]
@NiklasBr I was not able to reproduce this (using windows 10 - Postman v7.19.1).
It would really help if you could share a screen recording showing the issue.

Here's what I tried (I am using default copy/paste keyboard keys -- ctrl-c and ctrl-v)
![postman-windows-copy-paste-640](https://user-images.githubusercontent.com/5207331/75514386-61d32380-5a1d-11ea-8b51-0e13c87a336a.gif)

**Comment by NiklasBr** [comment]
> You are up to date! Postman v7.19.1 is the latest version.
> Windows 10 Pro, version 1909

This is what I get when trying CKEditor (4.12), or Slack (4.3.4):

```html
<p><span style="color: rgb(80, 80, 80); font-family: OpenSans, Helvetica, Arial, sans-serif; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: pre-wrap; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">https://removed.domain/rest/v1/articles/read?market=se&amp;lastmodified=2020-02-28T09:00:00</span></p>
```

**Comment by NiklasBr** [comment]
Also: Pasting into Word or Outlook will also apply Opensans and Helvetica formatting to the string - not nice. :/

**Comment by NiklasBr** [comment]
@harryi3t will this be fixed soon? It still happens in 9.1.5

**Comment by DannyDainton** [comment]
Unable to reproduce this issue in the latest version of the platform. If this is still an issue that you're facing we can look at reopening it again.

**Comment by NiklasBr** [comment]
@DannyDainton, we might need to re-open this.

> Postman v9.22.2

1. Select all content of the field
2. Copy
3. Paste into an online editor [such as CK Editor](https://ckeditor.com/docs/ckeditor5/latest/examples/builds-custom/full-featured-editor.html):
4. Look at the source:

![bild](https://user-images.githubusercontent.com/279826/176460992-0c744c87-e712-474c-a88e-19a2d30baac8.png)

**Comment by DannyDainton** [comment]
Hey @NiklasBr 

I've copied the contents of the URL and pasted it into the editor that you linked and it's just displaying the same URL.

<img width="1130" alt="image" src="https://user-images.githubusercontent.com/17089546/176464397-8f342fc7-7f18-43d0-8806-7d21c49d4c9b.png">

Could you do a quick recording showing this behaviour, please?

**Comment by NiklasBr** [comment]
It looks like you mixed up step 3 and 4. 

Here is a recording, sorry about the cat's shaking the video.

https://user-images.githubusercontent.com/279826/176503742-190d89e6-4557-400b-aa93-d21393f1ce98.mp4

**Comment by NiklasBr** [comment]
@DannyDainton

**Comment by DannyDainton** [comment]
I was finally able to reproduce this but only when using the 3rd party editor you linked. You mentioned that this was causing issues when attaching items to Slack, etc can you provide examples of that behaviour, please?

**Comment by NiklasBr** [comment]
Thanks for re-opening this. Here are two examples from Outlook and Teams.

![image](https://user-images.githubusercontent.com/279826/179726741-9e186de8-8f24-4722-b0bb-591fec8ac387.png)

Note that the default font in Outlook is Verdana 11pt.

Then paste the copied GET text it detects it as a link (which is understandable), but changes the font to `Helvetica 12px` (9pt is about 12px) which is set in the `style` element of the copied text, compare to styles seen in https://github.com/postmanlabs/postman-app-support/issues/8101#issuecomment-1170051643:

![image](https://user-images.githubusercontent.com/279826/179727117-90237cf2-a60e-4bad-82ac-752d8c53b7db.png)

If I were to first paste the GET text into something like Notepad or other plain text field, the formatting is removed and *then* paste into Outlook the result is what I expect and want to happen when I copy directly from Postman:
![image](https://user-images.githubusercontent.com/279826/179727682-ae0852ee-d050-41d0-bb36-ec39f8995789.png)

<hr>

You can see the same behaviour in e.g. Teams where font styling such as Helvetica and fontsize remains when pasting directly from Postman.

Direct copy-paste into Teams:
![image](https://user-images.githubusercontent.com/279826/179728276-85d9eaa7-ef58-4fb8-b2d7-4a9729fd4c1b.png)

Copy-paste via Notepad and then into Teams:
![image](https://user-images.githubusercontent.com/279826/179728427-81bcc6a9-08e3-472c-b16d-622c9979657c.png)

<hr>

Short version: I expect pasted content to behave as plain-text, not rich text.

**Comment by NiklasBr** [comment]
@DannyDainton, any news? You asked for examples above

**Comment by NiklasBr** [comment]
@DannyDainton, any news? You asked for examples above

**Comment by NiklasBr** [comment]
@DannyDainton, any news? You asked for examples above

**Comment by NiklasBr** [comment]
@DannyDainton, any news? You asked for examples above

**Comment by DannyDainton** [comment]
@NiklasBr There is no news from me. The reason I asked for examples, was due to the lack of reproducible information in your originally raised issue. 

This is now in the hands of the team to triage and prioritise against all the open issues that we currently have on this repo and our internal tracker, any updates will be posted, if/when there is additional information to share.

**Comment by NiklasBr** [comment]
I don't agree that the initial post lacked reproducible information. Can you tell me what I could do better next time?

**Comment by DannyDainton** [comment]
A more targeted set of steps to reproduce the issue and the extra information provided (The video showing the issue, [screenshots and context here](https://github.com/postmanlabs/postman-app-support/issues/8101#issuecomment-1188885171) ) after asking for it, would have made it easier for us to triage it earlier. 

When you're describing an issue that you can see in front of you, it's better to over share the information to paint the picture of the problem. The more visual you can make it the better. I was only able to reproduce the issue after seeing the video.
